docs/topics/modules/glossary.adoc

```diff
@@ -38,40 +38,40 @@ backlog:: A backlog is a list of work items that have not been triaged and assig
 // endterm
 
 // term: e8d54bf3-f89e-46e5-86f7-4af6475863b0, en_EN
-bug:: Defect that causes unexpected behavior in the software.
+bug:: A defect that causes unexpected behavior in the software and that needs to be fixed.
 // endterm
 
 // term: 1694e637-2f9b-40ec-8fa8-a22472850ff9, en_EN
 companion dependency:: Based on the analysis of your stack, {osio} suggests additional dependencies that can add value to your stack.
 // endterm
 
 // term: 23c322f1-53b1-4286-b524-37ab58124823, en_EN
-experience:: Experience describes the envisioned user experience in the product to actualize a parent work item. Each parent work item can have multiple experiences.
+experience:: An experience is a way to address a scenario, fundamental, or a set of papercuts, each of which often have multiple realizing experiences. Often written as a demonstration script, it describes the ideal end-user work flow for realizing one or more of the associated value propositions.
+// endterm
+
+// term: 9ee45d71-1efe-4fec-b1bb-ac92563327a6, en_EN
+experiences:: Work item type group that focuses on the experience and value proposition work item types that address the scenarios.
```

Comment:
@rkratky while I agree with your point. We did not exactly conclude  what should be the plan of action to have infotips for such sections. Do we want to create a separate module for such cases?
Also this UX is based on the SDD template after many rounds of discussions between the team and PM. We could probably make our suggestions in the kabal tomorrow, and while we are on the topic, if you can take a look at the scrum template too. I am quite sure we would be discussing on similar lines for those WITs too, so we might as well talk about it too.
Typing through phone as I am commuting, pls excuse any typos

Reply:
@Preeticp This is a pickle... For the time being, I suggest we keep these "additional" terms commented out, so that they get pulled into the JSON, but they don't get shown in the docs. I'll try to play with this now.